catalyst/runners/runner.py

```diff
@@ -32,9 +37,26 @@
 from catalyst.utils.data import get_loaders_from_params
 from catalyst.utils.misc import maybe_recursive_call, set_global_seed
 
+if SETTINGS.amp_required:
+    from catalyst.engines import AMPEngine, DataParallelAMPEngine, DistributedDataParallelAMPEngine
+
 
 def _get_default_engine(fp16: bool = False, ddp: bool = False):
```

Comment:
btw, could we move it to `catalyst/utils/torch.py` as `get_engine`?
like https://github.com/catalyst-team/catalyst/blob/dev/catalyst/utils/torch.py#L170

Reply:
sure